tests/tests/test-issuers.spec.js

```diff
@@ -0,0 +1,146 @@
+const ISSUER_NAME = `test-issuer-${Math.floor(Math.random() * 9999) + 1000}`;
+const SECRET_NAME = `issuer-secret-${Math.floor(Math.random() * 9999) + 1000}`;
+
+context('Test Issuers', () => {
+  before(() => {
+    cy.loginAndSelectCluster();
+    cy.goToNamespaceDetails();
+
+    cy.getLeftNav()
+      .contains('Configuration')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Secrets')
+      .click();
+
+    cy.getIframeBody()
+      .contains('Create Secret')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Secret Name"]:visible')
+      .type(SECRET_NAME);
+
+    cy.getIframeBody()
+      .contains('.fd-dialog__content button', 'Create')
+      .click();
+
+    cy.url().should('match', new RegExp(`/secrets/details/${SECRET_NAME}$`));
+  });
+
+  it('Create an issuer', () => {
+    cy.getLeftNav()
+      .contains('Issuers')
+      .click();
+
+    cy.getIframeBody()
+      .contains('Create Issuer')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Issuer Name"]:visible')
+      .type(ISSUER_NAME);
+
+    cy.getIframeBody()
+      .contains('Select issuer type')
+      .filter(':visible')
+      .click();
+
+    cy.getIframeBody()
+      .contains('CA')
+      .filter(':visible')
+      .click();
+
+    cy.task('getNamespace').then(ns => {
+      cy.getIframeBody()
+        .find('[placeholder="Select Namespace"]:visible')
+        .type(ns);
+
+      cy.getIframeBody()
+        .contains('li', ns)
+        .click();
+    });
+
+    cy.getIframeBody()
+      .find('[placeholder="Select name"]:visible')
+      .type(SECRET_NAME);
+
+    cy.getIframeBody()
+      .contains('li', SECRET_NAME)
+      .click();
+
+    cy.getIframeBody()
+      .contains('.fd-dialog__content button', 'Create')
```

Comment:
It should be 
`    .find('[role=dialog]')
      .contains('button', 'Create')`